bin/nib-ls

```diff
@@ -180,6 +184,22 @@ def proc_file(f, opts):
     else:
         row += [ '' ]
 
+    if opts.header_fields:
+        # signals "all fields"
+        if opts.header_fields == '*':
+            # TODO: might vary across file types, thus prior sensing would be needed
+            header_fields = h.keys()
+        else:
+            header_fields = opts.header_fields.split(',')
```

Comment:
Maybe filter up here for invalid or empty keys?  Raise error instead of passing silently?